capstone/src/test/java/com/step902020/capstone/UserDatastoreTest.java

```diff
@@ -248,6 +289,89 @@ public void testOrganizationDeleteEvent() throws URISyntaxException {
     // Check organization after delete ("postcondition")
     Organization postResult = authRestTemplate.getForObject("/get-organization", Organization.class);
     assertEquals(
-            "Delete event error", 0, postResult.getEvents().size());
+            "Delete event error", 2, postResult.getEvents().size());
+  }
+
+  @Test
+  public void testGetIndividualRecommendedEventsNoRecommendedEventsReturned() throws URISyntaxException {
+    List<Individual> users = this.individualRepository.findByUniversity(expectedIndividual.getUniversity());
+    Recommender mockRecommender = Mockito.mock(Recommender.class);
+    when(mockRecommender.recommend(expectedIndividual, users, u -> u.getSavedEvents(), 2)).thenReturn(new ArrayList<Event>());
+    IndividualController unitUnderTest = new IndividualController();
+    unitUnderTest.setRecommender(mockRecommender);
+    // getting the actual result
+    Event[] result = authRestTemplate.getForObject("/get-recommended-events-individual?count=2", Event[].class);
+    assertEquals("Wrong number returned", 2, result.length);
+  }
+
+  @Test
+  public void testGetIndividualRecommendedEventsOneRecommendedEventsReturned() throws URISyntaxException {
+    List<Individual> users = this.individualRepository.findByUniversity(expectedIndividual.getUniversity());
+    Recommender mockRecommender = Mockito.mock(Recommender.class);
+    List<Event> recommendedEvents = new ArrayList<Event>();
+    recommendedEvents.add(expectedEvent2);
+    when(mockRecommender.recommend(expectedIndividual, users, u -> u.getSavedEvents(), 2)).thenReturn(recommendedEvents);
+    IndividualController unitUnderTest = new IndividualController();
+    unitUnderTest.setRecommender(mockRecommender);
+    // getting the actual result
+    Event[] result = authRestTemplate.getForObject("/get-recommended-events-individual?count=2", Event[].class);
```

Comment:
How do you know these two recommended events contain one "recommended" and one "filler" event?